app/controllers/provider_interface/provider_users_controller.rb

```diff
@@ -120,5 +145,12 @@ def find_provider_user
     rescue ActiveRecord::RecordNotFound
       render_404
     end
+
+    def find_provider_permissions_model
+      ProviderPermissions.find_by(
```

Comment:
This will raise `ActiveRecord::RecordNotFound`:

```suggestion
      ProviderPermissions.find_by!(
```

Reply:
Makes sense, thanks